clients/telegrafs/telegrafs.go

```diff
@@ -0,0 +1,173 @@
+package telegrafs
+
+import (
+	"context"
+	"errors"
+	"io/ioutil"
+	"os"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+	"github.com/influxdata/influx-cli/v2/pkg/influxid"
+)
+
+type Client struct {
+	clients.CLI
+	api.TelegrafsApi
+}
+
+type TelegrafParams struct {
+	clients.OrgParams
+
+	Desc string
+	File string
+	Name string
+
+	Id  string
+	Ids []string
+}
+
+type telegrafPrintOpts struct {
+	graf  *api.Telegraf
+	grafs *api.Telegrafs
+}
+
+func (c Client) Telegrafs(ctx context.Context, params *TelegrafParams) error {
+	if params.OrgID == 0 || params.Id == "" {
```

Comment:
Needs a check on `params.Org` as well? (or maybe it's `params.OrgName`)

Reply:
You can also check `!params.Valid()` instead of checking for `0`

Reply:
Looks like in the original code, it was getting the OrgID from a function by passing in both the ID and Name, and `if id == 0`, then it would lookup by name. Do we have functionality somewhere globally for this, or should I implement it myself?